requirements/constraints.txt

```diff
@@ -36,9 +36,6 @@ django-oauth-toolkit<=1.3.2
 # Will be updated once we update python-dateutil package
 matplotlib<3.4.0
 
-# tests failing for pymongo==3.11
```

Comment:
Add new pin here to avoid any major release.

Reply:
Done